src/main/java/com/app/bookstore/service/cart/item/CartItemServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.app.bookstore.service.cart.item;
+
+import com.app.bookstore.dto.cart.item.CartItemResponseDto;
+import com.app.bookstore.dto.shopping.cart.ShoppingCartRequestDto;
+import com.app.bookstore.exception.EntityNotFoundException;
+import com.app.bookstore.mapper.CartItemMapper;
+import com.app.bookstore.model.CartItem;
+import com.app.bookstore.model.ShoppingCart;
+import com.app.bookstore.repository.book.BookRepository;
+import com.app.bookstore.repository.cart.item.CartItemRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CartItemServiceImpl implements CartItemService {
+    private final CartItemRepository cartItemRepository;
+    private final CartItemMapper cartItemMapper;
+    private final BookRepository bookRepository;
+
+    @Override
+    public CartItemResponseDto updateCartItemById(Long cartItemId, int quantity) {

```

Comment:
If it is CartItemService, than we can avoid using CardItem in methods.
```suggestion
    public CartItemResponseDto updateById(Long id, int quantity) {

```

Reply:
Make sure to simplify it everywhere.